docs/integration/c.md

```diff
@@ -3,4 +3,5 @@ title: Using MF2 with C
 sidebar_title: C
 ---
 
-TODO
+There is currently no pure C implementation of MF2.
```

Comment:
Even if the implementation is in C++, can you not use ICU from C? I don't know much about this, so please tell me if I am wrong :)

Reply:
I believe there has to be at least some glue code to allow calling C++ code in ICU from C, which doesn't currently exist, but I haven't looked into it much. I can experiment with it after doing the docs for C++ and Java!